webplugin/js/app/media/media-service.js

```diff
@@ -1,6 +1,28 @@
 Kommunicate.mediaService = {
     browserLocale:
-       window.navigator.language || window.navigator.userLanguage || 'en-us',
+        window.navigator.language || window.navigator.userLanguage || 'en-US',
+    isAppleDevice: function () {
+        var isIOSDevice = /iPhone|iPad|iPod/i.test(navigator.userAgent);
+        var isMacSafari = /^((?!chrome|android).)*safari/i.test(navigator.userAgent);
+
+        return isIOSDevice || isMacSafari
+    },
+    endSttExplicitly: function (lastListeningEventTime, recognizingDone, recognition) {
+        const checkTimeSTT = 1000;
```

Comment:
what is checkTimeSTT?

Reply:
need to change the variable name, earlier it was used as a interval time as well as for millisecond to sec conversion
updated it to
const MILLISECOND_TO_SEC = 1000;